apps/understack-workflows/eventsource-openstack/argo-rabbitmq.yaml

```diff
@@ -3,30 +3,23 @@ apiVersion: rabbitmq.com/v1beta1
 kind: User
 metadata:
   name: argo
-  namespace: openstack
 spec:
-  tags:
-  - management  # available tags are 'management', 'policymaker', 'monitoring' and 'administrator'
-  - policymaker
   rabbitmqClusterReference:
     name: rabbitmq  # rabbitmqCluster must exist in the same namespace as this resource
     namespace: openstack
-  importCredentialsSecret:
-    name: argo-rabbitmq-password
 ---
 apiVersion: rabbitmq.com/v1beta1
 kind: Permission
 metadata:
   name: argo-to-ironic-permission
-  namespace: openstack
 spec:
   vhost: "ironic"
   userReference:
     name: "argo"  # name of a user.rabbitmq.com in the same namespace; must specify either spec.userReference or spec.user
   permissions:
-    write: ".*"
-    configure: ".*"
-    read: ".*"
+    write: "^$"
+    configure: "^$"
```

Comment:
have you tested this? IIRC removing configure permission resulted in EventSource failure, but cannot remember what was it exactly

Reply:
So it should only result in an EventSource failure if we're changing anything from the config that creates that. It's currently deployed and not reporting any errors.

```
Status:
  Conditions:
    Last Transition Time:  2024-08-30T21:03:17Z
    Status:                True
    Type:                  Deployed
    Last Transition Time:  2024-08-30T21:03:17Z
    Status:                True
    Type:                  SourcesProvided
```